### ISSUE TITLE
Shairport not playing as a service

### ISSUE THREAD

**Issue by ghost** [mikebrady/shairport-sync#684]
Hello,

I just installed shairport-sync on my Raspberry Pi 3. I wanted to use an external USB DAC because I didn't read that there is a solution for the poor sound quality of the built-in audio. However I already have the USB DAC so I want to use it. Unfortunately no sound is playing what I put the shairport as output device on my apple devices. I tried with shairport-sync -vvv when the service is stopped and everything works fine. This makes me think I have a problem with the permissions of the shairport-sync user. I added him to the pi group, but this didn't help.

I will be thankful if someone offers a solution.

Best Regards
Backo

**Comment by backo123** [comment]
Changed the user and group in the service cons file and it works like a charm.

**Comment by mikebrady** [comment]
It’s good that you got your problem solved, but the problem should not have happened. When you were building Shairport Sync, did you to the `$ sudo make install` step, which creates the `shairport sync` user and group, if necessary?

**Comment by backo123** [comment]
Yes I didn't miss this step.

Yesterday when I finally got it to work I saw that the shairport-sync user might have been desabled. I use Webmin to administrate whatever is possible trough some user-friendly GUI. So it is possible the shairport-user may have been disabled while I was trying, but I wasn't in the mood to check if this was the case :). I can try this evening when I return home and let you know.

**Comment by mikebrady** [comment]
Thanks. It would be interesting alright, as I have no experience of using the GUIs; maybe there is something there we need to be aware of.

**Comment by backo123** [comment]
To be honest I didn't follow the guide for installation you wrote here, but this one - [](https://pimylifeup.com/raspberry-pi-airplay-receiver/)
I followed it up to step 8, but on the step for creation of the user and the group I got messages that they exist, which makes me think I tried installation before, but I didn't finished it or deleted the installation without deleting the user. Maybe somehow the user got disabled. I am really bad at remembering what I have done to the raspberry in the last few months.

I will try if it will work with the user enabled and all, and get back to you with the result.

**Comment by mikebrady** [comment]
Super, thanks. BTW, I just emailed the write of the guide, alerting them to the fact that their Step 6 isn't needed anymore.

**Comment by backo123** [comment]
Unfortunately after being sure that the shairport-user is enabled, the service didn't manage to play anything. I returned my configuration with the pi user. 
As I said earlier, most probably there is a problem with my OS as I have installed and uninstalled quite a lot of things. If I am going to do a clean install I will start with shairport-sync form the beginning and if the problem persists I will come back to you. For now it is important that it works.

Thanks a lot for your interest in my issue. Keep it up!

Best Regards
Backo
